### ISSUE TITLE
PyCall fails to precompile - LoadError of `wstring`

### ISSUE THREAD

**Issue by JonnyCBB** [stevengj/PyCall.jl#317]
I can no longer get PyCall to precompile on Windows 10 using Julia version v0.5.0-rc3.   

Here is the error:
```
julia> using PyCall
INFO: Precompiling module PyCall...
ERROR: LoadError: LoadError: LoadError: `wstring` has been moved to the package LegacyStrings.jl:
Run Pkg.add("LegacyStrings") to install LegacyStrings on Julia v0.5-;
Then do `using LegacyStrings` to get `wstring`.

 in include_from_node1(::String) at .\loading.jl:426
 in macro expansion; at C:\Users\jonathan\.julia\v0.4\PyCall\src\startup.jl:46 [inlined]
 in anonymous at .\<missing>:?
 in include_from_node1(::String) at .\loading.jl:426 (repeats 2 times)
 in macro expansion; at .\none:2 [inlined]
 in anonymous at .\<missing>:?
 in eval(::Module, ::Any) at .\boot.jl:234
 in process_options(::Base.JLOptions) at .\client.jl:239
 in _start() at .\client.jl:318
while loading C:\Users\jonathan\.julia\v0.4\PyCall\src\..\deps\deps.jl, in expression starting on line 4
while loading C:\Users\jonathan\.julia\v0.4\PyCall\src\startup.jl, in expression starting on line 42
while loading C:\Users\jonathan\.julia\v0.4\PyCall\src\PyCall.jl, in expression starting on line 37
ERROR: Failed to precompile PyCall to C:\Users\jonathan\.julia\lib\v0.5\PyCall.ji
 in compilecache(::String) at .\loading.jl:505
 in require(::Symbol) at .\loading.jl:364
```
I installed the `LegacyStrings` and loaded it using `using LegacyStrings` as suggested in the error message, but the error message I get after trying to import PyCall again is the same as before.

**Comment by stevengj** [comment]
I can't reproduce your problem .... this should have been fixed ages ago, in cd80a6e5fac0e3e70f72fb9bef87bfdbf354ab6b

What is `Pkg.status("PyCall")`?  Are you using an old version of PyCall?

**Comment by JonnyCBB** [comment]
Sorry to bother you. It turns out that I hadn't had PyCall installed in my .julia v0.5 directory (only in v0.4). I just needed to `Pkg.add("PyCall")` again. I removed `LegacyStrings` and it still works fine. It's all sorted
